packages/story-utils/src/__tests__/getStories.ts

```diff
@@ -0,0 +1,168 @@
+import mock from 'mock-fs';
+import getAllStories from '../getStories';
+
+afterEach(() => {
+  mock.restore();
+});
+
+test('finding stories', () => {
+  mock({
```

Comment:
Doh, mock-fs is working locally, but not on CI, for some reason 😞

Reply:
Looks like it works on Mac, but not Ubuntu or Windows. I think that mock-fs actually isn't intended to work with `require` calls.